app/services/foreman_seeder.rb

```diff
@@ -0,0 +1,62 @@
+require 'digest'
+
+class ForemanSeeder
+  FOREMAN_INTERNAL_KEY = 'database_seed'.freeze
+
+  attr_reader :seeds
+
+  def initialize
+    @seeds = (foreman_seeds + plugin_seeds).sort_by { |seed| seed.split("/").last }
+  end
+
+  def foreman_seeds
+    Dir.glob(Rails.root + 'db/seeds.d/*.rb')
+  end
+
+  def plugin_seeds
+    Foreman::Plugin.registered_plugins.collect do |name, plugin|
+      engine = (name.to_s.tr('-', '_').camelize + '::Engine').constantize
+      Dir.glob(engine.root + 'db/seeds.d/*.rb')
+    rescue NameError => e
+      Foreman::Logging.exception("Failed to register plugin #{name}", e)
```

Comment:
This: this isn't really registration.

Reply:
The wording aint the best per say

Reply:
I later saw that it was just moved from somewhere else. Might be a good chance to improve but also fine to keep it.